### ISSUE TITLE
BUG-005 - WAVE: Missing form labels

### ISSUE THREAD

**Issue by JosephBanham** [lunarcroissant/react-challenge-1-window-news#5]
The form/s on the page lack labels - This is important due to the way screenreaders interpret these elements.

For instance, if I had two forms without labels next to each other, a screenreader may read them as:
"Form. Search here."
"Form. Search for products."

What would be the difference between the two? This is what labels fill. Labels should be unique, and should detail the purpose of the form.

"Form. Web search bar. Search here."
"Form. My favourite items search bar. Search for products."

**Comment by lunarcroissant** [comment]
Done :)
